styles/youtube/catppuccin.user.css

```diff
@@ -382,6 +382,9 @@
       --yt-endpoint-visited-color: @accent-color !important;
       color: @accent-color !important;
     }
+    .badge-style-type-live-now-alternate.ytd-badge-supported-renderer[aria-label="PREMIERE"] {
+      color: @surface0 !important;
+    }
```

Comment:
The premier badge was added here, https://github.com/catppuccin/userstyles/pull/897/files#diff-c08ba2f30a7396e0d274fcbc429a1616b4dd4373f815d47f2b082ef52dbf4928R698. Do you mind changing this value instead,

Reply:
changed it